pkg/apiserver/api/v1/meta.proto

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+package kubecarrier.api.v1;
+option go_package = "v1";
+
+import "google/protobuf/timestamp.proto";
+
+message ObjectMeta {
+  string uid = 1;
+  string name = 2;
+  google.protobuf.Timestamp creationTimestamp = 3;
+  google.protobuf.Timestamp deletionTimestamp = 4;
+  string resourceVersion = 5;
+  map<string, string> labels = 6;
+  map<string, string> annotations = 7;
+  int64 generation = 8;
```

Comment:
I use `int64` instead of `uint64` since kubernetes uses `int64` and want to keep consistent with it.

Reply:
:+1: Could you also please add a `string account = 9;`?
As this field is essentially our `Namespace`